source/features/hide-diff-signs.tsx

```diff
@@ -3,4 +3,7 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-void features.addCssFeature(import.meta.url, [pageDetect.hasCode], 'has-rgh-inner');
+void features.addCssFeature(import.meta.url, [
+	pageDetect.hasCode,
+	() => pageDetect.isEditingFile() && !location.href.endsWith('.md'),
```

Comment:
https://github.com/refined-github/refined-github/issues/4244

Reply:
> What happens if this feature is enabled on .md? If it doesn't break anything, I'd just drop this check

It doesn't, it's just that the feature is useless here.